SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -401,6 +402,11 @@ void UGlobalStateManager::SetCanBeginPlay(bool bInCanBeginPlay)
 
 	bCanBeginPlay = bInCanBeginPlay;
 	NetDriver->Connection->SendComponentUpdate(GlobalStateManagerEntityId, &Update);
+
+	if (bCanBeginPlay)
+	{
+		NetDriver->World->GetWorldSettings()->SetGSMReadyForPlay();
```

Comment:
Why does it have to be set again here?

Reply:
Yeah, can probably move both of these such that it's only called in TriggerBeginPlay. We just need to make sure it's set on the authoritative server, and the non-authoritative server.